plugin/src/main.rs

```diff
@@ -17,37 +15,40 @@ struct CliArgs {
     /// If unset, defaults to the default namespace in the current context.
     #[clap(global = true, long, short = 'n')]
     namespace: Option<String>,
-
-    #[clap(flatten)]
-    args: cli_utils::CliArgs,
 }
 
 impl CliArgs {
     async fn args() -> Self {
         let mut arg = CliArgs::parse();
-        if let Some(ns) = arg.namespace.clone() {
-            arg.args.args.namespace = ns;
-        } else {
-            let client = Client::try_default().await.expect("Client init failed");
-            let ns = client.default_namespace().to_string();
-            arg.args.args.namespace = ns;
+        let ns = match arg.namespace {
+            Some(ref namespace) => namespace.to_string(),
+            None => {
+                let client = Client::try_default().await.expect("Client init failed");
+                client.default_namespace().to_string()
+            }
+        };
+        match arg.operations {
+            cli_utils::Operations::Mayastor(ref mut operations) => {
+                operations.cli_args.namespace = ns
+            }
+            cli_utils::Operations::LocalpvLvm(ref mut operations) => {
+                operations.cli_args.namespace = ns
+            }
+            cli_utils::Operations::LocalpvZfs(ref mut operations) => {
+                operations.cli_args.namespace = ns
+            }
+            cli_utils::Operations::LocalpvHostpath(ref mut operations) => {
+                operations.cli_args.namespace = ns
+            }
         }
         arg
     }
 }
 
-impl Deref for CliArgs {
-    type Target = plugin::CliArgs;
-
-    fn deref(&self) -> &Self::Target {
-        &self.args
-    }
-}
-
 #[tokio::main]
 async fn main() {
     let cli_args = CliArgs::args().await;
-    let _tracer_flusher = cli_args.init_tracing();
+    //let _tracer_flusher = cli_args.init_tracing();
```

Comment:
Why is this commented?

Reply:
I will remove it. We dont need tracing for plugin.

Reply:
We don't we need it?
What if we need to debug it?

Reply:
I tried,  println!() works. That should be enough right?

Reply:
humm, you can't compare tracing/opentelemetry to a println :)

Reply:
Done, we init tracing for Mayastor commands.